server/routes/index.ts

```diff
@@ -19,13 +19,15 @@ import printReportRoutes from './printReport'
 import assaultRoutes from './assault'
 import selectAssociatedPrisonerRoutes from './selectAssociatedPrisoner'
 import selectAssociatedStaffRoutes from './selectAssociatedStaff'
+import offenceCodeDecisionsRoutes from './offenceCodeDecisions'
 
 import { Services } from '../services'
 
 export default function routes(
   router: Router,
   { placeOnReportService, locationService, prisonerSearchService, reportedAdjudicationsService, userService }: Services
 ): Router {
+  router.use('/offence-code', offenceCodeDecisionsRoutes({ placeOnReportService }))
```

Comment:
Don't know if this is the url to use - I think we'll need to discuss these when we start wiring up journeys

Reply:
Look Ok to me! i suppose `offence-code-selection` might be a bit more appropriate?!